openedx/features/course_experience/static/course_experience/js/course_outline_factory.js

```diff
@@ -23,6 +24,16 @@
                         break;
                     }
                 });
+
+                $('a:not([href^="#"])').click(function(event) {
```

Comment:
It does seem very general, but I can't think of a great central place to put it. For the sake of expediency, I think this is fine here.